motionplan/metrics.go

```diff
@@ -39,9 +40,9 @@ func CombineMetrics(metrics ...Metric) Metric {
 	return cm.combinedDist
 }
 
-// orientDist returns the arclength between two orientations.
+// orientDist returns the arclength between two orientations in degrees.
 func orientDist(o1, o2 spatial.Orientation) float64 {
-	return math.Sqrt(spatial.QuatToR3AA(spatial.OrientationBetween(o1, o2).Quaternion()).Norm2())
+	return utils.RadToDeg(spatial.QuatToR4AA(spatial.OrientationBetween(o1, o2).Quaternion()).Theta)
```

Comment:
Is this a 1:1 change?

Reply:
This was what was asked for by Product in the scope doc.

In terms of what's being measured, yes, that's the same, just expressed a different way.